sections/05_treat.md

```diff
@@ -146,7 +153,9 @@ information from protein-protein or protein-compound interaction networks
 [@doi:10.1186/1758-2946-5-30 @doi:10.1021/ci500340n
 @doi:10.1186/s12859-015-0845-0].
 
-Menden et al [@doi:10.1371/journal.pone.0061318] used a shallow neural network
+`TODO: shallow NN? Is that in scope?`
```

Comment:
We may have included some shallow networks in other sections.  It is slightly out of scope, but I think here it gives context about the later work.

@enricoferrero you added this reference.  Any thoughts?